psi4/src/psi4/libmints/dipole.h

```diff
@@ -46,14 +46,8 @@ class Molecule;
  *
  * Use an IntegralFactory to create this object. */
 class DipoleInt : public OneBodyAOInt {
-    //! Obara and Saika recursion object to be used.
+    // This is to disappear when l2 dipole derivatives are worked out
```

Comment:
See #2414 for details. I think the easiest way is to implement analytic derivatives of multipole moment integrals using McMurchie-Davidson rather than working this out in l2. @andysim please confirm my statement 😄

Reply:
Max is right - we have a pilot implementation of arbitary order multipoles, and their derivatives.  This O-S recursion object will disappear when the M-D integrals are added :)